k8s/src/main/scala/io/buoyant/k8s/istio/MixerClient.scala

```diff
@@ -25,14 +26,17 @@ private[telemetry] object MixerClient {
     mapper.readValue[Seq[String]](yaml).asInstanceOf[Seq[String]]
   }
 
-  private[telemetry] def indexOf(dict: Seq[String], str: String): Int =
+  private[this] def indexOf(dict: Seq[String], str: String): Int =
     -1 * (dict.indexOf(str) + 1)
 
-  private[telemetry] def mkReportRequest(): ReportRequest = {
+  // TODO: http1-only for now
+  private[istio] def mkReportRequest(req: Request, rsp: Option[Response]): ReportRequest = {
```

Comment:
I think there's additional information we want to pass in here such as request duration, dst, and possibly other params from the endpoint stack.  We could add all of those things as parameters to this method or we could move the body of this method into the istio logger so it has direct access to that information.